Dragonflight/TheAzureVault/Locales/esES.lua

```diff
@@ -3,7 +3,7 @@ if not L then return end
 if L then
 	L.book_of_translocation = "Libro de traslado"
 	L.custom_on_book_autotalk = "Hablar automáticamente"
-	--L.custom_on_book_autotalk_desc = "Instantly proceed to the next area when talking to Books of Translocation."
+	L.custom_on_book_autotalk_desc = "Accede directamente al siguiente área al hablar con el boss de translocación."
```

Comment:
Is "boss" the right translation here? The NPC you talk to is https://www.wowhead.com/es/npc=194602/libro-de-traslado.

Reply:
My bad, in spanish we use some english words in very informal and relaxed situations and i went wrong. Now that i see which NPC you are talking about we can solve this replacing with "Accede directamente al siguiente área al hablar con el jefe de traslado" or use the NPC name "Accede directamente al siguiente área al hablar con libro de traslado". Personally i think the second option is more correct. Should i change it? Or are you going to handle it?

Reply:
My bad i read boss instead of books. To solve it replace with "Accede directamente al siguiente área al hablar con los libros de traslado."

Reply:
If you want it in french too; "Avance directement à la prochaine surface après avoir parlé aux livres de transposition."

Reply:
Go ahead and update it in your PR.

If you want to do French translations too that's also appreciated, you can open a second PR for that.